src/controller/Observer.java

```diff
@@ -0,0 +1,18 @@
+package controller;
```

Comment:
You can add a line explanation about this design pattern

Reply:
Done

Reply:
V